### PR TITLE
Disable Dynamic Trace File

### PR DESCRIPTION
**Pull request by yinggeh**
Disable dynamic trace file location, aka `--trace_file` API.

### CODE REVIEW

**Review by nnshah1 — APPROVED** [review]
Thanks for the fast work! LGTM

File: src/tracer.cc
Comment (indrajit96) [inline src/tracer.cc:1]: If we don't support update of trace_file now 
Should we clean up "filepath_specified" from the tracer..cc altogether? As filepath_specified_ will always be false?

Can take up in another PR as well as the current changes are for a PSIRT

Comment (yinggeh) [reply src/tracer.cc:1]: No. It' not. Trace file can be specified when launching tritonserver. In this case filepath_specified is true.

**Review by rmccorm4 — APPROVED** [review]
LGTM - is there a separate PR for client/PA tests in the works?

**Pull request merged**